lib/logstash/outputs/influxdb.rb

```diff
@@ -29,10 +32,10 @@ class LogStash::Outputs::InfluxDB < LogStash::Outputs::Base
   config :port, :validate => :number, :default => 8086
 
   # The user who has access to the named database
-  config :user, :validate => :string, :default => nil, :required => true
+  config :user, :validate => :string, :default => nil
 
   # The password for the user who access to the named database
-  config :password, :validate => :password, :default => nil, :required => true
+  config :password, :validate => :password, :default => nil
```

Comment:
Same here, you don't need to specify `:default => nil` since nil is implicit default.